api/filters.go

```diff
@@ -429,11 +432,14 @@ func (api *API) updateFilterDimension(w http.ResponseWriter, r *http.Request) {
 		"dimension_name": dimensionName,
 	}
 
-	dim := model.Dimension{
-		Options: []string{},
+	updateReq := updateFilterDimensionRequest{
```

Comment:
Unless there is another type of request to differentiate from just `req` is fine/preferable